components/brave_news/browser/resources/Sidebar.tsx

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+import Icon from '@brave/leo/react/icon';
+import { color, effect, font, radius, spacing } from '@brave/leo/tokens/css';
+import * as React from 'react';
+import styled, { css } from 'styled-components';
+import { FeedType, useInspectContext } from './context';
+import Card from './feed/Card';
+import { isPublisherEnabled } from './shared/api';
+
+const DEFAULT_SHOW_COUNT = 4;
+
+const PAD_LEFT = '34px';
+
+const Container = styled(Card)`
+  padding: 16px 8px;
+  align-self: flex-start;
+  position: sticky;
+  top: ${spacing.xl};
+`
+
+const Heading = styled.h3`
+  font: ${font.primary.default.semibold};
+  color: var(--bn-card-color-0.75);
+  margin: 0;
+
+  padding-left: ${PAD_LEFT};
+`
+
+const CustomButton = styled.button <{ selected?: boolean, faint?: boolean, bold?: boolean }>`
+  padding: 8px 8px 8px ${PAD_LEFT};
+
+  display: block;
+  outline: none;
+  border-radius: ${radius.m};
+  border: none;
+  background: none;
+  text-align: left;
+  width: 100%;
+
+  color: ${p => `rgba(var(--bn-text-base), ${p.faint ? 0.25 : 1})`};
```

Comment:
cc @petemill I haven't worked out a complete theming story yet - everything is very custom.